src/jvmMain/kotlin/baaahs/jvmImpl.kt

```diff
@@ -54,13 +55,13 @@ class RealFs(private val basePath: Path) : Fs {
     }
 }
 
-actual fun logMessage(level: String, message: String, exception: Throwable?) {
+actual fun log(id: String, level: String, message: String, exception: Throwable?) {
+    val logger = LoggerFactory.getLogger(id)
```

Comment:
Does this log to a log file now? Or is it still logging in the browser console?

Reply:
Logs in the browser still go to console. The jvm Pinky now logs through the normal Byzantine maze of Java logging frameworks.